aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -222,6 +241,21 @@ Tensor& div_(Tensor& self, Scalar other) {
   return self.div_(wrapped_scalar_tensor(other)); // redispatch!
 }
 
+Tensor remainder(const Tensor& self, Scalar other) {
+  Tensor other_tensor = wrapped_scalar_tensor(other);
+  return native::remainder(self, other_tensor.toType(self.scalar_type()));
```

Comment:
the scalar type conversion looks like it retains the same behavior, but I don't think it's actually what we want long term

(it differs from NumPy, for example, CC @mruberry).

I think we should leave it as is, but could you just put a comment that reads something like:
"FIXME: do normal type promotion a-la NumPy"

Reply:
Comment added.